src/App.js

```diff
@@ -1,7 +1,16 @@
-import React from "react";
+import Mockman from "mockman-js";
+import { Route, Routes } from "react-router-dom";
+import { LandingPage } from "./pages";
 
 const App = () => {
-  return <div>App</div>;
+  return (
+    <>
+      <Routes>
```

Comment:
Suggestion - This is fine for now. But later on, when the routes increase, you could move this to a separate component (say NavRoutes or WebsiteRoutes) and then render it here.

Reply:
Okay Bharati, I will take care of this.